### DISCUSSION TITLE
Can I disable tweaks for specific games?

### DISCUSSION THREAD

**Discussion by ghost** [tkashkin/GameHub#485] [Q&A] [answered]
So if some games don't work with specific tweaks,can I disable those tweaks just for them?

**Comment by tkashkin** [comment, marked as answer]
Yes. To configure Tweaks for a game:
* Right click a game and select `Tweaks` (for current releases: `dev`, `master`)
* Open game's properties and select `Tweaks` tab (for the next release in `refactoring` branch)

**Reply by ghost** [reply]
I mean I can disable tweaks that are enabled globally for all gamehub games,but I can't enable tweaks that are globally disabled there,is there a way at all to do that?
